input/papers/2016/2016arXiv160908671H/tev-000095.yaml

```diff
@@ -0,0 +1,26 @@
+paper_id: 2016arXiv160908671H
+source_id: 95
+
+data:
+  livetime: 13.4 # year 2012
+#  livetime: 65.3 year 2011, earlier observations have already been published
+#  significane: 3, 5, 7, 9 mentioned in paper fig 4
+
+pos:
+  ra: {val: 17.226} # nominal centre of the SNR, see tab 1, what about centred 17h13m25.2s see fig 2?
+  dec: {val: -39.76} # nominal centre of the SNR, see tab 1, wht about centred -39d46m15.6s see fig 2?
+
+morph:
+  type: point # because of PSF? or Gaussian see p.3 or Shell see p.3
+#  sigma: {val: 0.036 same as superior angular resolution p.3?}
+#  pa: same as resulting centre point, see p.3?
+
+spec:
+  type: ecpl # what does this mean?
+
+  norm: {val: 20.2, err: 0.8} # see tab 3, F_0
```

Comment:
Instead of three comments for each parameter, I think one comment on the line before, saying where the values come from (which section of table in the paper) would be better than duplicating that info in the comment for each parameter 3 times.